src/Reflector.php

```diff
@@ -87,18 +87,27 @@ final class Reflector
     /**
      * @param array<string,string>|null $symbols
      * @param array<string,string>      $source
-     * @param string[]                  $missingSymbols
+     * @param array<string, int>        $missingSymbols
      */
     private static function initSymbolList(?array &$symbols, array $source, array $missingSymbols): void
     {
         if (null !== $symbols) {
             return;
         }
 
+        $excludingSymbols = array_keys(
+            array_filter($missingSymbols, static function ($version) {
```

Comment:
missing imports for:

- `array_filter`
- `PHP_VERSION_ID`

Note that this is configurable on phpStorm if you use it (look for "import function")

Also minor, but I tend to prefer this CS:

```php
array_keys(
    array_filter(
        $missingSymbols,
        static function (int $version): bool {
             return PHP_VERSION_ID < $version;
        }
    )
);
```

Reply:
Next todo: add a linter job for phpcs in GitHub Action 😄

Reply:
imports and cs fixed